db/migrations/000005_add_org_id_column.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE connections
+    ADD org_id varchar(100);
```

Comment:
For now, let's set the org_id as a varchar(20).  That might still be a bit of overkill but it should give us plenty of room to grow.